cmd/generate/flag.go

```diff
@@ -0,0 +1,41 @@
+package generate
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagApp          = "app"
+	flagDir          = "dir"
+	flagInstallation = "installation"
+	flagVersion      = "version"
+)
+
+type flag struct {
+	App          string
+	Dir          string
+	Installation string
+	Version      string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.App, flagApp, "", `Name of an application to generate the config for (e.g. "kvm-operator").`)
+	cmd.Flags().StringVar(&f.Dir, flagDir, "", "Directory containing configuration. If empty, contents of giantswarm/config will be used.")
```

Comment:
Let me know if this is acceptable.

Reply:
How about passing a branch instaed?

Reply:
I thought we can just point it to a dir, regardless of it being git cloned or not. We can update flags when we implement pulling from git anyways.

Reply:
That would require yet another implementaiton. When pulling from git we we would be able to find issues with the original implementaiton (at a cost of extra pushing step). If it isn't convincing then let's proceed with --dir.

Reply:
Changed to `Branch`